ci/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+set -e -u -x -o pipefail
+
+cd "$(dirname "$0")/../"
+
+ci/bootstrap.sh
+.shared-ci/scripts/prepare-unity.sh
+
+source .shared-ci/scripts/pinned-tools.sh
+source .shared-ci/scripts/profiling.sh
+
+UNITY_PROJECT_DIR="$(pwd)/workers/unity"
+
+markStartOfBlock "$0"
+
+markStartOfBlock "Building UnityClient"
+
+# TODO - run unity stuff
```

Comment:
Are these TODOs to be resolved in this PR?

Reply:
They have already been done 😂